lib/model/store.dart

```diff
@@ -185,6 +189,11 @@ class PerAccountStore extends ChangeNotifier {
 
   // TODO lots more data.  When adding, be sure to update handleEvent too.
 
+  final Map<String, RealmDefaultExternalAccount> realmDefaultExternalAccounts;
+
+  // TODO call [CustomProfileFieldsView.dispose] in [dispose]
+  final CustomProfileFieldsView customProfileFields;
```

Comment:
Should we have `zulipVersion` as a sort of special field that belongs first in this section? The other sections are alphabetically ordered. I can see three options here:

The version is special enough to be first (and the rest of the items then ordered):
```
  final String zulipVersion; // TODO get from account; update there on initial snapshot
  final List<CustomProfileField> customProfileFields;
  final int maxFileUploadSizeMib; // No event for this.
  final Map<String, RealmDefaultExternalAccount> realmDefaultExternalAccounts;
```

or the simpler datatypes are before the "richer" datatypes: 
```
  final String zulipVersion; // TODO get from account; update there on initial snapshot
  final int maxFileUploadSizeMib; // No event for this.
  final List<CustomProfileField> customProfileFields;
  final Map<String, RealmDefaultExternalAccount> realmDefaultExternalAccounts;
```

or a plain refactor of the sort order:
```
  final List<CustomProfileField> customProfileFields;
  final int maxFileUploadSizeMib; // No event for this.
  final Map<String, RealmDefaultExternalAccount> realmDefaultExternalAccounts;
  final String zulipVersion; // TODO get from account; update there on initial snapshot
```

Reply:
I like the second option as well. In general I tend to prefer a logical or semantic order rather than alphabetical, when it's manageable to do so.

For example, `streams` goes before `subscriptions` because the information of what streams exist is logically prior to the information of which streams our user is subscribed to, and to the other information about the user's relationship to the streams (like are they pinned, what color are they shown with, etc.).

Here I guess that would further point to putting `realmDefaultExternalAccounts` just before `customProfileFields`, because the former is part of the information one needs for interpreting the latter.